site/_docs/operating/cartridges.md

```diff
@@ -7,11 +7,28 @@ permalink: /docs/operating/cartridges/
 
 Once you [generate a project](/adop-docker-compose/docs/operating/projects), you will be able to load [cartridges](/adop-docker-compose/docs/architecture/cartridges/).
 
+Prerequisite
+
+Please follow below instruction if you are loading cartridge from a private repository to enable SSH key permission from your Jenkins to the private repository:
+
+1.	Navigate to the following Jenkins URL (http://<public-ip>/jenkins/userContent)
```

Comment:
Only minor issue is that the url appears as:
http:///jenkins/userContent

Can this please be updated so that <public-ip> is actually displayed?